content/en/docs/configuration/acme/dns01/google.md

```diff
@@ -122,3 +130,104 @@ spec:
 ```
 
 For more details about `Certificates`, see [Usage](../../../../usage/).
+
+## GKE Workload Identity
+
+If you are deploying cert-manager into a [Google Container Engine (GKE)
+cluster](https://cloud.google.com/kubernetes-engine/) with workload identity
+enabled, you can leverage workload identity to avoid creating and managing
+static service account credentials. The [workload identity
+how-to](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity)
+provides more detail on how workload identity functions, but briefly workload
+identity allows you to link a Google service accounts (GSA) to Kubernetes
+service accounts (KSA). This GSA/KSA linking is two-way, i.e., you must
+establish the link in GCP _and_ Kubernetes. Once configured, workload identity
+allows Kubernetes pods running under a KSA to access the GCP APIs with the
+permissions of the linked GSA. The workload identity how-to also provides
+detailed instructions on how to enable workload identity in your GKE cluster.
+The instructions in the following sections assume you are deploying cert-manager
+to a GKE cluster with workload identity already enabled.
+
+### Link KSA to GSA in GCP
+
+The cert-manager component that needs to modify DNS records is the pod created
+as part of the cert-manager deployment. The [standard methods for deploying
+cert-manger to Kubernetes](../../../installation/kubernetes.md) create the
+cert-manager deployment in the cert-manager namespace and its pod spec specifies
+it runs under the cert-mananger service account. To link the GSA you created
+above to the cert-manager KSA in the cert-manager namespace in your GKE cluster,
+run the following command.
+
+```bash
+$ gcloud iam service-accounts add-iam-policy-binding \
```

Comment:
Have you tested if this stays when upgrading cert-manager? I see you apply an annotation below which could be deleted on upgrade

Reply:
The `gcloud` commands persist when upgrading cert-manager.

Reply:
If you use `kubectl apply` to upgrade, then the annotation added later should remain as well. If you delete and recreate, then you would need to add the annotation again.

Reply:
okay, we recommend to use applies so that would work, also with the helm tag it solves that too :smile: